reports/project-groupby-status.ts

```diff
@@ -24,7 +24,7 @@ export function getDefaultConfiguration(): any {
     'flag-in-progress-days': 21,
     'wip-limit': 2,
     limits: {},
-    'status-label-match': '(?:green|yellow|red)'
+    'status-label-match': '(^green$|^yellow$|^red$)'
```

Comment:
regex was little too loose here and got a false positive

Reply:
we could have just changed the yaml but I opted for the default being better ...